rswl_plot.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+
+def lookup2d(data):
+
+    data_width = data[:, [0, 1, 2]]
+    data_length = data[:, [0, 1, 3]]
+    data_sigmaw = data[:, [0, 1, 4]]
+    data_sigmal = data[:, [0, 1, 5]]
+
+    width = data_width[:, 2].reshape(
+                                     (len(np.unique(data_width[:, 0])),
+                                      len(np.unique(data_width[:, 1]))
+                                      ))
+
```

Comment:
oh no! .. no I understand...  `data[:, [0, 1]]` are shapes .. and they should all be identical .. ok this is probably still under construction here. forget my previous comment...

Reply:
OK, I have changed the table to be structured array with named columns. It is much more transparent.